blocks/header/header.css

```diff
@@ -463,29 +503,15 @@ header nav .nav-explore {
   background-color: var(--udexColorBlue7);
   height: 3.938rem;
   width: 100%;
-  padding: 0 var(--udexSpacer24);
 }
 
-/* header nav[aria-expanded='false'] .nav-explore {
-  display: none;
-} */
-
-header nav .nav-explore ul {
-  list-style: none;
-  padding: unset;
-}
-
-header nav .nav-explore ul > li > a {
-  font-family: var(--udexTypographyNavigationRegularXLFontFamily);
-  font-weight: var(--udexTypographyNavigationRegularXLFontWeight);
-  color: var(--udexColorNeutralWhite);
-  font-size: var(--udexTypographyNavigationRegularXLFontSize);
-  line-height: var(--udexTypographyNavigationRegularXLLineHeight);
-  white-space: nowrap;
-  text-decoration: none;
+@media (width < 980px) {
```

Comment:
Usually `>=` queries should suffice and the smaller screen sizes are the defaults. Is it possible to achieve without the `<`